articles/downloading_data.md

```diff
@@ -0,0 +1,143 @@
+---
+title: "Downloading data"
+layout: article
+excerpt: This tutorial will cover the best practices of downloading a file, directory and managing where files are stored
```

Comment:
Managing where files are stored locally? Cuz my files are stored in the Synapse.